### DISCUSSION TITLE
Issues with Student Pack proof

### DISCUSSION THREAD

**Discussion by TheDevMinerTV** [community/community#57525] [GitHub Education]
### Hi everyone,

Heya,

I'm on my last two days of Student Pack right now and wanted to reverify. Though, now when I take the picture I just get the error message `You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio`.
I was using my "school certificate" to prove that I'm in my school.
<img src="https://github.com/community/community/assets/29845135/0ba17576-be5f-4d38-b828-47e464fd8facg"  height="300">

I can't even hide my full name. I'm trans and don't want people to dead name me all the time just because my full name is on my GH profile, not even mentioning that I don't want to have my full name out there.

Is there any way to avoid this privacy issue?

PS: This verification process has only been getting worse over the last three years...
PPS: Support wasn't helpful, it took them 23 days to reply with "Sorry lol, ask in the community".
PPPS: I can select every type of proof but the error message tells me that the "image should one of the following: Dated school ID", then why have the dropdown anyway?

### A suggestion...(optional)

Disable this check

Update 07.06.2023: I tried my school ID and got to the "Thanks for submitting" page. That one got declined as well, the problem is probably that the date on German school IDs is written on the back and I can only upload one image...

Update 09.06.2023: I tried changing my full name in GitHub, reverifying and then changing it back, which also didn't work...

**Comment by chriskthomas** [comment]
> Update 09.06.2023: I tried changing my full name in GitHub, reverifying and then changing it back, which also didn't work...

Wow. Did it fail with the same error message?

**Reply by TheDevMinerTV** [reply]
Yep, exactly the same error message. It's hilarious how bad this process is...

**Comment by TheDevMinerTV** [comment]
My student pack expired and no-one cared. Thanks Microsoft.

I can't close the discussion with the correct resolution, so I'm not gonna.

**Comment by samus-aran** [comment]
Hi @TheDevMinerTV 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.

We are extremely proud that the GitHub Global Campus has provided millions of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 
However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification.

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information as it's only required upon each new submission. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

If you do want to try again we would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

**Reply by ghost** [reply]
I still don't understand.

> “One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.”

Can't a bad actor just change their GitHub account information too? It's not their own details, so they don't care. I hope your “internal team” comes to this same realisation.

If they are particularly headstrong about asking for your legal name, it should be part of the application form instead.

**Reply by TheDevMinerTV** [reply]
I'm already out of school now. I honestly don't know why my full name is required to be on my **PUBLIC** profile, just make it an input in the actual form, then you can even OCR it or something.



PS: The GitHub team is slow to respond, as always.
PPS: For the German "Schülerausweis", you have to upload both sides, because one contains the name, the other one the stamp (atleast in Saxony).

**Reply by lavender-sys** [reply]
@samus-aran This feels awfully dismissive. As a trans student whos still closeted irl, I don't want to be revealing my deadname to everyone online. Doing so would very much open me up to harassment.

**Reply by Clechay** [reply]
Same, I tried everything and still no student pack for me
